### DISCUSSION TITLE
harmony

### DISCUSSION THREAD

**Discussion by Flu09** [theislab/scvelo#1213] [Q&A]
General question.  I am working on several datasets that have been already integrated by harmony. I am wondering if scvelo will work on such integrated object? or if you have recommendations on the integration method preceding the use of scvelo?

**Comment by jjia1** [comment]
I don't see why it would cause any problems. Harmony is only used for batch correction and generates a new set of PCs that are supposed to reduce batch effects. If you don't actually end up using the harmony PCs, then I can't see it causing any problems. Even if you do use those PCs, I don't see why it would cause any issues as it should be better than using PCs generated from regular PCA.
